src/main/resources/hudson/plugins/scm_sync_configuration/ScmSyncConfigurationPlugin/help/manualSynchronizationIncludes_fr.properties

```diff
@@ -1,20 +1,13 @@
-List\ of\ ant-like\ includes\ allowing\ to\ specify\ additionnal\ sync-ed\ files\ with\ repository=\
-  Liste de chemin \u00e0 la Ant, permettant de sp\u00e9cifier des fichiers suppl\u00e9mentaires qui seront synchronis\u00e9s avec le repository
-Includes\ should\ be\ case\ sensitive\ paths\ starting\ from\ your\ JENKINS_HOME\ directory,\ without\ /\ in\ the\ beginning;\ Use\ of\ wildcards=\
-  Les chemins doivent \u00eatre sensibles \u00e0 la casse, bas\u00e9s depuis votre r\u00e9pertoire JENKINS_HOME, et ne d\u00e9marrant pas par un /; L''utilisation des wildcards
+List\ of\ ant-like\ includes\ allowing\ to\ specify\ additional\ files\ to\ synchronize\ with\ the\ repository=Liste de chemin \u00E0 la Ant, permettant de sp\u00E9cifier des fichiers suppl\u00E9mentaires qui seront synchronis\u00E9s avec le repository
```

Comment:
The only change here is actually this key, which needs to match the new English text in `manualSynchronizationIncludes.jelly` above.

I hadn't noticed the other changes; looks like eclipse automatically reformatted the properties upon saving. Sorry about that.

D'ailleurs, cela ne devrait pas être "Liste de chemin**s** ..."?